R/subset_wuse.R

```diff
@@ -17,13 +17,13 @@
 #' areas <- "10" # NA uses all areas
 #' area.column <- "STATECODE"
 #' 
-#' w.wuse.sub <- subset_wuse(w.use, data.elements,area.column,areas)
+#' w.use.sub <- subset_wuse(w.use, data.elements,area.column,areas)
 subset_wuse <- function(w.use, data.elements, area.column, areas=NA) {
   # subset w.use on basis of areas and select the data elements of interest
   if (all(is.na(areas))){
-    w.use.sub <- w.use[,c("YEAR",area.column,data.elements)]
+    w.use.sub <- w.use[,c("YEAR","STATECOUNTYCODE",area.column,data.elements)]
```

Comment:
Just to chime in here a bit.  ```data.element``` is usually reserved for water use category codes~ ie Public Supply, Irrigation, etc.  Don't think we want to throw ```STATECOUNTYCODE``` in there. A generic "Area" would be best.   Just my 2 cents.